libs/logging/src/log_event_ids.ts

```diff
@@ -74,14 +74,14 @@ export enum LogEventId {
   ClearedBallotData = 'clear-ballot-data-complete',
   OverridingMarkThresholds = 'override-mark-threshold-init',
   OverrodeMarkThresholds = 'override-mark-thresholds-complete',
-  DownloadedScanImageBackup = 'download-backup-scan-images',
+  SavedScanImageBackup = 'saved-backup-scan-images',
```

Comment:
I see this discrepancy was already here so feel free to ignore! But could be nice to make the variable name and string match up:

```
SavedScanImageBackup = 'saved-scan-image-backup'
```

Reply:
tweaked